rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -3271,7 +3288,7 @@ def test_latest_upgrade_correctness(user_data_dir):
     assert tables_after_creation - tables_after_upgrade == set()
     assert views_after_creation - views_after_upgrade == set()
     new_tables = tables_after_upgrade - tables_before
-    assert new_tables == {'cowswap_orders', 'gnosispay_data'}
+    assert new_tables == {'cowswap_orders', 'gnosispay_data', 'evmtx_topics_index'}
```

Comment:
hmmm? The cowswap orders and gnosispay data are new tables in this release?

Reply:
indeed nice that you noticed